src/Ledger/GovernanceActions.lagda

```diff
@@ -131,23 +221,63 @@ record GovProposal : Set where
         prevAction  : NeedsHash action
         anchor      : Anchor
 \end{code}
-\begin{code}[hide]
+\caption{Governance action proposals and votes}
+\label{defs:governance-votes}
+\end{figure*}
+To submit a governance action to the chain one must provide a deposit which will be returned when
+the action is finalized (whether it is \defn{ratified} or has \defn{expired}).
+The deposit amount will be added to the \defn{deposit pot}, similar to stake key deposits. It will also be
+counted towards the stake of the reward address it will be paid back to, to not reduce the submitter's
+voting power to vote on their own (and competing) actions.
+\\[4pt]
+\textbf{Remarks}.
+\begin{enumerate}
+\item A motion of no-confidence is an extreme measure that enables Ada holders to revoke the power that has been granted to the current constitutional committee.
+\item A \emph{single} governance action might contain \emph{multiple} protocol parameter updates. Many parameters are inter-connected and might require moving in lockstep.
+\end{enumerate}
 
+\begin{code}[hide]
 instance
   _ = +-0-commutativeMonoid
   unquoteDecl DecEq-GovRole = derive-DecEq ((quote GovRole , DecEq-GovRole) ∷ [])
   unquoteDecl DecEq-Vote    = derive-DecEq ((quote Vote    , DecEq-Vote)    ∷ [])
   unquoteDecl DecEq-VDeleg  = derive-DecEq ((quote VDeleg  , DecEq-VDeleg)  ∷ [])
 \end{code}
-\caption{Governance actions and votes}
-\label{defs:governance}
-\end{figure*}
 
+See Section~\ref{sec:ratification} for more on the ratification process.
+
+\subsection{Protocol parameters and governance actions}
+\label{sec:protocol-parameters-and-governance-actions}
+Recall from Section~\ref{sec:protocol-parameters}, the \AgdaRecord{PParam} type is a record containing
+parameters used in the Cardano ledger, grouped according to the general purpose that each parameter serves.
+(See Figure~\ref{fig:protocol-parameter-declarations}.)
+\begin{itemize}
```

Comment:
We don't need to repeat that list here. The only thing that's relevant in this section is that there are multiple groups, not really their details.